src/main/scala/mesosphere/marathon/core/group/impl/GroupManagerImpl.scala

```diff
@@ -49,6 +50,7 @@ class GroupManagerImpl(
   private[this] val root = LockedVar(initialRoot)
 
   private[this] val dismissedDeploymentsMetric = Metrics.counter(ServiceMetric, getClass, "dismissedDeployments")
+  private[this] val groupUpdateSizeCounter = Metrics.minMaxCounter(ServiceMetric, getClass, "queueLength")
```

Comment:
I am not strong on this, but I would propose to use the suffix `Metric` on all metric-related variables. So `groupUpdateSizeMetric`

Reply:
I don't like that much... but I am not strong on this either, so we probably need someone third :D

Reply:
it's been `Metric` at the end so far, correct? Either rename them all to smth. else or keep it.

Reply:
ok ... to me it's like naming integer properties `AbcInt` and String properties 'AbcString' but whatever :) ... changed